### PR TITLE
rpm generation

### PR DESCRIPTION
**Pull request by pabeni**
this changes add an 'rpm' target to the main makefile

details in the individual commit messages

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -109,17 +114,29 @@ dummy_test: $(dummy_test-objs)
 
 all: $(binaries)
 
+dist: distclean
+	mkdir rushit-$(VERSION)
+	rsync -Cavz --exclude 'rushit-$(VERSION)' . rushit-$(VERSION)
+	tar -czf $(DIST_ARCHIVES) rushit-$(VERSION)
```
Comment (jsitnicki) [inline Makefile:120]: Can we use `git archive` to replace the above 3 steps?

File: rushit.spec [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Version:	%{mk_version}
+Release:	1%{?dist}
+Summary:	scriptable performance tool
+
+Group:		Applications/Internet
+License:	ASL 2.0
+URL:		https://github.com/jsitnicki/rushit/
+Source0:	rushit-%{version}.tar.gz
+
+BuildRequires:	libcmocka-devel
```
Comment (jsitnicki) [inline rushit.spec:13]: If we don't run the tests (`make check`) as part of the build then `libcmocka-devel` should not be needed.

File: rushit.spec [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+make %{?_smp_mflags}
+
+%install
+install -p -D -t $RPM_BUILD_ROOT/%{_bindir}/ tcp_stream
+install -p -D -m 0644 -t $RPM_BUILD_ROOT/%{_datadir}/rushit/ scripts/*.lua
+install -p -D -m 0644 -t $RPM_BUILD_ROOT/%{_docdir}/rushit/ README.md README.neper.md
+
+%files
+%defattr(-,root,root)
+%{_bindir}/tcp_stream
```
Comment (jsitnicki) [inline rushit.spec:32]: There is also `tcp_rr` binary. Could you please add it?

**Comment by pabeni** [comment]
still need to fix copr integration

**Pull request closed without merging**